src/MongoDB.Driver.Legacy/Builders/IndexKeysBuilder.cs

```diff
@@ -566,6 +615,19 @@ public override BsonDocument ToBsonDocument()
             return _indexKeysBuilder.ToBsonDocument();
         }
 
+        /// <summary>
+        /// Sets wildcard keys to index. If the wildcard names collection is empty, the generated key will be for the whole collection.
+        /// </summary>
+        /// <param name="memberExpressions">The member expressions.</param>
+        /// <returns>
+        /// The builder (so method calls can be chained).
+        /// </returns>
+        public IndexKeysBuilder<TDocument> Wildcard(params Expression<Func<TDocument, object>>[] memberExpressions)
+        {
+            _indexKeysBuilder = _indexKeysBuilder.Wildcard(GetElementNames(memberExpressions).ToArray());
+            return this;
+        }
+
```

Comment:
Done

Reply:
I'm not sure that passing `$**` instead of null is the best option here since we already have validation on the empty string in line 282. So, I think it's better to have such logic in one place.